tests/test_client.py

```diff
@@ -0,0 +1,167 @@
+import pytest
+import testinfra
+import time
+import subprocess
+
+
+@pytest.fixture(scope='session')
+def katello_client(request, entities, registration_hostname):
+    image_id = subprocess.check_output(['podman', 'images', '-q', '--filter', f"reference={request.param}"])
+    assert image_id
+
+    container_id = subprocess.check_output(['podman', 'run', '-v', '/dev/log:/dev/log', '--systemd=true', '-dt', request.param, '/sbin/init']).decode().strip()
```

Comment:
Rather than passing `/sbin/init`, shouldn't we set `CMD` in the Dockerfiles we maintain? Also, `--systemd=true` is the default value. I'd also prefer long options in scripts. So something like this:
```suggestion
    container_id = subprocess.check_output(['podman', 'run', '--volume', '/dev/log:/dev/log', '--detach', '--tty', request.param]).decode().strip()
```